tutorials/3d/lights_and_shadows.rst

```diff
@@ -48,30 +50,54 @@ Lights can optionally cast shadows. This gives them greater realism (light does
 not reach occluded areas), but it can incur a bigger performance cost.
 There is a list of generic shadow parameters, each also has a specific function:
 
--  **Enabled**: Check to enable shadow mapping in this light.
--  **Color**: Areas occluded are multiplied by this color. It is black by default, but it can be changed to tint shadows.
--  **Bias**: When this parameter is too small, self shadowing occurs. When too large, shadows separate from the casters. Tweak to what works best for you.
--  **Contact**: Performs a short screen-space raycast to reduce the gap generated by the bias. Contact shadows are only available when using the GLES3 backend.
--  **Reverse Cull Faces**: Some scenes work better when shadow mapping is rendered with face-culling inverted.
+- **Enabled:** Check to enable shadow mapping in this light.
+- **Opacity:** Areas occluded are darkened by this opacity factor. Shadows are
+  fully opaque by default, but this can be changed to make shadows translucent
+  for a given light.
+- **Bias:** When this parameter is too small, self shadowing occurs. When too
+  large, shadows separate from the casters. Tweak to what works best for you.
+- **Contact:** Performs a short screen-space raycast to reduce the gap generated
+  by the bias. Contact shadows are only available when using the GLES3 backend.
+- **Reverse Cull Face:** Some scenes work better when shadow mapping is rendered
+  with face-culling inverted.
 
```

Comment:
Several things about the above section.
- Contact is no longer a parameter at least in beta 4
- Transmittance bias, normal bias and blur are all missing from this list.